slide/src/test/ui/output_form/s_expression/unwrap_parens_var.slide

```diff
@@ -0,0 +1,19 @@
+!!!args
+--parse-only
+-o s-expression
+!!!args
+
+===in
+(a)
+===in
+
+~~~stdout
+(a)
```

Comment:
Shouldn't this just be `a` or am I missing something?

Reply:
We stop at the parsing phase, see `--parse-only` in the args above